test/modules/standard/Random/bradc/testGetNth.chpl

```diff
@@ -13,7 +16,8 @@ for i in 1..n {
   if (r1 != r2 || r2 != r3) {
     writeln("mismatch at #", i, ": ", r1, " ?= ", r2, " ?= ", r3);
   } else {
-    writeln("#", i, " = ", r2);
+    // Don't print the #s in this test; just check for a match
+    //writeln("#", i, " = ", r2);
```

Comment:
True. I wanted to make the tests in this directory apply to either RNG. We now have test/modules/standard/Random/ferguson/npb-rand-check.chpl and pcg-rand-check.chpl that verify the RNG output (and in the PCG case compare against numbers generated by the C version).
I could put the .good back, adding a 2nd one for PCG, but I don't personally like to have .good files that are there just to warn us something changed (even if the change is not necessarily wrong). In this case, I'd prefer to have fewer cases that print out the random numbers. If you have a strong preference about this test in particular, I'll put the numbers back in.